openlibrary/solr/update_work.py

```diff
@@ -552,6 +552,7 @@ def add_list(name, values):
         add('type', 'work')
         add('seed', BaseDocBuilder().compute_seeds(w, editions))
         add('title', w.get('title'))
+        add('title_sortable', w.get('title'))
```

Comment:
If that's not possible, this will need to be wrapped in a

```py
if get_solr_next():
    add('title_sortable', w.get('title'))
```

So that it doesn't try to add a non-existent field to our current solr!